### ISSUE TITLE
Name of repository 

### ISSUE THREAD

**Issue by kelu124** [echopen/medicotechnical#1]
Alors, technicomedical ou medicotechnical ? :)

**Comment by Olivierff** [comment]
Pour ma part, plutôt, intuitivement medicotechnique mais à vous de me dire.

Le mar. 12 avr. 2016 à 14:29, kelu124 <notifications@github.com> a écrit :

> Alors, technicomedical ou medicotechnical ? :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/echopen/medicotechnical/issues/1>
>
-- 
Olivier de Fresnoye
@OFresnoye
+33 (0) 6 47 74 86 89

**Comment by benchoufi** [comment]
je dirai à choisir medicotechnical ;)
btw, a-t-on arrêter un schéma de nomination homogène ? underscore, casse, affixation. genre med_tech, medTech, ou autre

**Comment by kelu124** [comment]
@BiVi est dessus =)

**Comment by Bivi** [comment]
Oui, je vais faire une proposition sur la syntaxe des refs. Ce sera un truc du genre :
- Nature entité : 3 lettres maj
- séparateur : tiret (qui pourrait poser pb si on l'utilise en nom de variable)
- le nom : succession de mots en minuscules séparés par des "_" seuls caractères autorisés [0-9a-z_]. En ordonnant les mots du plus général au plus precis (comme ça le tri alpha regroupera correctement des symboles). On evitera les abbréviations sauf si elles sont reconnus dans tout le système et dans ce on utilisera uniquement ces abbréviations partout. Ex : 
  * MDL-realtime_controller_cc_trinket
  * MDL-realtime_controller_stepper_cortex_m4
  * ITF-0f_3_3v : on pourrait dors et déjà fixer les n° de pistes.
  * ITF-10_gnd : n° hexa, centré sur 0x10 (donc une capacité de 32 pins de 0x00 à 0x1f)
  * ITF-11_5v : on pourrait aussi numéroter en alpha centré sur la lettre "m"
-
